nvtabular/workflow/workflow.py

```diff
@@ -165,6 +169,10 @@ def output_node(self):
     def _input_columns(self):
         return self.graph._input_columns()
 
+    def get_subgraph(self, subgraph_name):
+        subgraph = self.graph.subgraph(subgraph_name)
+        return Workflow(subgraph.output_node)
```

Comment:
Since this method returns a `Workflow`, perhaps `get_subworkflow` would match this return type more closely?

Reply:
If we did go with `get_subworkflow` or similar for this method, then the property above `subraphs` might also need to change to something like `subworkflows` or `subworkflow_names`?